vulcanai2/models/cnn.py

```diff
@@ -41,18 +49,17 @@ def _create_network(self):
             else:
                 pass
 
-        if self._input_network and isinstance(self._input_network, DenseNet):
+        if self._input_network and isinstance(self._input_network, DenseUnit):
```

Comment:
No this should be `DenseNet`, i think we should add import of `DenseNet` as well to this file.

Reply:
ok